From c9836c658bb8ac32dec823b629396475e76e5582 Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rstoyanov@fedoraproject.org>
Date: Thu, 27 Jan 2022 22:13:33 +0000
Subject: [PATCH 111/249] ci: set continue-on-error for cross-compile

Running cross compile tests with Debian unstable sometimes
fails due to missing or outdated packages.

Signed-off-by: Radostin Stoyanov <rstoyanov@fedoraproject.org>
---
 .github/workflows/cross-compile.yml | 16 ++++++++++++----
 1 file changed, 12 insertions(+), 4 deletions(-)

diff --git a/.github/workflows/cross-compile.yml b/.github/workflows/cross-compile.yml
index 461a6e618..be8e7f09c 100644
--- a/.github/workflows/cross-compile.yml
+++ b/.github/workflows/cross-compile.yml
@@ -6,18 +6,26 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
+      fail-fast: false
       matrix:
+        experimental: [false]
         target: [
           armv7-stable-cross,
-          armv7-unstable-cross,
           aarch64-stable-cross,
-          aarch64-unstable-cross,
           ppc64-stable-cross,
-          ppc64-unstable-cross,
           mips64el-stable-cross,
-          mips64el-unstable-cross
         ]
+        include:
+          - experimental: true
+            target: armv7-unstable-cross
+          - experimental: true
+            target: aarch64-unstable-cross
+          - experimental: true
+            target: ppc64-unstable-cross
+          - experimental: true
+            target: mips64el-unstable-cross
 
     steps:
     - uses: actions/checkout@v2
-- 
2.35.1

